server/controllers/scripts.js

```diff
@@ -0,0 +1,136 @@
+import knexModule from 'knex';
+import bookshelfModule from 'bookshelf';
+import scriptsService from '../db/services/scripts';
+import { development as devconfig } from '../../knexfile';
+import Script from '../db/models/scripts';
+// import bookshelfCreateTable from '../db/init';
+
+const knex = knexModule(devconfig);
+const bookshelf = bookshelfModule(knex);
+const ScriptModel = Script(bookshelf);
+
+export function saveNewScript(req, res, next) {
+  console.log(req.body, 'params in saveNewScript');
+  const params = {
+    name: req.body.name,
+    body: req.body.body,
+    description: req.body.description
+  };
+
+  scriptsService.saveNewScript(params, ScriptModel)
+    .then((script) => {
+      if (script) {
+        res.status(201).json(script);
+      } else {
+        next();
+      }
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(401).json({ message: 'Could not save script.' });
```

Comment:
Why are we returning a 401 here? In fact, other route methods don't seem to handle errors at all? What is the default behavior for express when a route method doesn't return a value or errors out?